test/spec/controllers/mediators.js

```diff
@@ -88,4 +97,19 @@ describe('Controller: MediatorsCtrl', function () {
     scope.mediators[1].endpoints.length.should.equal(2);
   });
 
-});
\ No newline at end of file
+  it('should calculate the lastHeartbeatStatus field based on the last heartbeat', function () {
+    createController();
+    httpBackend.flush();
+
+    scope.mediators[0].lastHeartbeatStatus.should.equal('success');
+    scope.mediators[1].lastHeartbeatStatus.should.equal('danger');
+  });
+
+  it('should set the uptimeDisplay field with a human friendly display string', function () {
+    createController();
+    httpBackend.flush();
+
+    scope.mediators[0].uptimeDisplay.should.equal('an hour');
+    scope.mediators[1].uptimeDisplay.should.equal('2 months');
+  });
+});
```

Comment:
Perhaps some test of the mediator details page would be good? Just to make sure things are added to the config object properly.